invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -154,6 +154,10 @@ class TitleTypeSchema(VocabularySchema):
     """Title type schema."""
 
 
+class DescriptionTypeSchema(VocabularySchema):
+    """Description type schema."""
+
```

Comment:
I'm starting to wonder why we started doing this inheritance... we are not adding anything. We could simply use the `VocabularySchema` below... right?

Reply:
I've noted down to discuss that at some point on standup. I agree with readability but I forsee many classes appearing that have nothing extra